### ISSUE TITLE
Luminosity ratio for the toggle buttons in legends becomes less than the required ratio on turning them off.

### ISSUE THREAD

**Issue by anubhutisaini12** [highcharts/highcharts#15159]
#### Expected behaviour
On turning the "Tokyo" OFF (any legend off), Luminosity of the color of button should be equal to or more than 3:1.

#### Actual behaviour
On turning the "Tokyo" OFF (any legend off), Its color change and luminosity of the color becomes 1.6:1 which is less than 3:1.

#### Live demo with steps to reproduce

https://www.highcharts.com/demo/column-basic
![image](https://user-images.githubusercontent.com/76141946/108160779-bc263200-710f-11eb-93a6-2ec2e02aec1e.png)

MAS Reference:
MAS 1.4.11 - Non-text Contrast

Browser: Edge chromium Version 87.0.634.0 (Official build) dev (64-bit),
Tool: Color contrast analyzer

**Comment by KacperMadej** [comment]
Hi @anubhutisaini12 

Thank you for reporting the problem.

As a workaround, the color could be adjusted in the chart configuration: https://api.highcharts.com/highcharts/legend.itemHiddenStyle

The lightest grey on white seems to be #949494 (ration >3.03) if we go by the non-text contrast - [demo](https://jsfiddle.net/BlackLabel/wqyngrh0/).
However, shouldn't this go by the text contrast of at least 4.5 (MAS 1.4.3)? Then the lightest grey is #767676 (>4.54) - [demo](https://jsfiddle.net/BlackLabel/wqyngrh0/1/).

fyi (a11y) @oysteinmoseng

**Comment by oysteinmoseng** [comment]
Yes, understanding the text would be considered essential to the operation of this control so this should be evaluated against text contrast - 4.5:1 in this case.

The symbol could still have a lighter color, as long as the text is intelligible. Example: https://jsfiddle.net/b65oenta/. I think this is a better solution - I am not a fan of having the difference between enabled/disabled be too small, that just makes it hard to determine which series are disabled.

We could consider adding an option like `legend.itemHiddenStyle.symbolColor` and then change the default values accordingly. What do you think @TorsteinHonsi?

**Comment by TorsteinHonsi** [comment]
Not sure... 

> I am not a fan of having the difference between enabled/disabled be too small, that just makes it hard to determine which series are disabled.

Indeed. And from an accessibility point of view, that is also unfortunate.

Actually I don't _like_ any of the options, but if it what it takes to conform to the requirements then I suppose we need to do it. Except `legend.itemHiddenStyle` is a CSSObject, so maybe we shouldn't put more custom properties in there.

**Comment by oysteinmoseng** [comment]
The ideal solution would be one that didn't rely on text dimming to indicate that a series is hidden (e.g. [checkboxes](https://api.highcharts.com/highcharts/legend.itemCheckboxStyle)), but it's hard to make that look as nice visually...

Thoughts on adding `legend.symbolHiddenStyle.color` vs `legend.symbolHiddenColor`?

**Comment by KacperMadej** [comment]
How about making the legend symbol inactive look like a checkbox?
Also, an italic font style might be an option - to increase the difference between enabled and disabled.
Demo: https://jsfiddle.net/BlackLabel/wqyngrh0/3/

IMHO Existing symbol options in legend are given one by one and not in a group, so `legend.symbolHiddenColor` seems to be more consistent.

**Comment by oysteinmoseng** [comment]
That checkbox-style idea is really cool @KacperMadej! We could combine with my solution and have a thinner (and brighter) outline for the checkbox if we wanted. Not sure about the italics, but what do you think @TorsteinHonsi?

**Comment by TorsteinHonsi** [comment]
I think with a 1px border and normal font-style it looks good: https://jsfiddle.net/highcharts/d8v6hpLb/. 

Now maybe we should have a 1px stroke for active elements too, so that the bounding box doesn't change when hovering. We may be able to avoid a visual diff nightmare by reducing the dot radius by 0.5px. And some CSS animations would be nice when changing the stroke and fill.

**Comment by anubhutisaini12** [comment]
Hi, 

Any update on this ?? Is development being done on this or is still in progress ?

**Comment by KacperMadej** [comment]
The status of the issue is marked with labels and could also be tracked through the GitHub projects page (Development-Flow).
Currently, the issue is planned for implementation but is not yet in progress. There is no ETA for the implementation at this moment.
